source/style.css

```diff
@@ -233,13 +234,11 @@ body.nerd-info :is(#nerd-info, #nerd-log) {
 
 #visualizer > div {
 	border: 1px solid #FFFFFF;
-	position: absolute;
 	left: 0;
 	width: 100%;
 }
 
 #visualizer .network-container {
-	top: 2%;
```

Comment:
After removing this, the visualization no longer centers to the screen correctly (when you press the centering button in the top right corner of the screen). See the old design in the left vs. your design on the right:
![image](https://user-images.githubusercontent.com/20535393/169879041-bad4f5b4-35d5-43a2-9f3a-41735cd89e4d.png)


I believe you can solve this by making the `#visualization` `height: 100vh` and then using `padding` to center the elements inside of it.